ffi/capi/js/examples/tinywasm/Makefile

```diff
@@ -33,8 +33,8 @@ target/wasm32-unknown-unknown/release/icu_capi_cdylib.wasm: FORCE
 		--target wasm32-unknown-unknown \
 		--release \
 		--package icu_capi_cdylib \
-		--no-default-features
-		--features buffer_provider,logging \
+		--no-default-features \
+		--features buffer_provider,logging,icu_capi/icu_decimal
```

Comment:
Enabling features on a dependency only works if the dependency is local, so this does not work as an example.

Reply:
Added a comment.

Reply:
"you should create your own cdylib crate" is not something that a non-Rust developer will be able to do. It wasn't required before, and it shouldn't be required now, we just need to set the same features as before: `--no-default-features --features buffer_provider,default_components[,logging]`.

Reply:
Actually, lets just migrate this example to compiled data to use default features. It also reduces the size and adds fallback which is currently missing.

Reply:
https://github.com/robertbastian/icu4x/commit/7cec5203e484b774df900077544c928f8253d892

Reply:
Okay but I prefer to merge the narrow fix first.

Reply:
The narrow fix would have been to use the features from my second comment, this actually changes the example